### DISCUSSION TITLE
Can't communicate with robot using Pi4

### DISCUSSION THREAD

**Discussion by gmptrr98** [iRobotEducation/create3_docs#220] [Questions] [answered]
Hello, I don't know why the robot can't seem to communicate with Pi4 at all. I followed the provided documentation, with the only difference that I also installed Ubuntu Mate as Gui using desktopify (https://github.com/wimpysworld/desktopify). ROS2 is running properly, the Pi4 turns on when attached to the robot and the driver's toggle is on usb. I'm using fastDDS both on the pi and create3.

What could be the problem?
Thank you very much for the help!

**Comment by justinIRBT** [comment, marked as answer]
Are you able to ping the robot over usb from the pi (`ping 192.168.186.2`)?  Did you connect your Create3 to the same wifi network as your raspberry pi, are you able to ping the robot wifi ip from the pi?

**Reply by gmptrr98** [reply]
I fixed the problem, it was due to wrong intendation when editing network-config file. 
Thank you!
